.github/workflows/events-nightly.yml

```diff
@@ -48,6 +48,18 @@ jobs:
           repository_url: https://test.pypi.org/legacy/
           verbose: true
 
+      # report failure to Slack
+      - name: Slack notification
+        if: failure() && github.event_name == 'schedule'
+        uses: ravsamhq/notify-slack-action@v1
+        with:
+          status: ${{ job.status }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          notification_title: 'Publish nightly package to test.pypi.org'
+          message_format: '{emoji} *{workflow}* {status_message}, see <{run_url}|detail>, cc: <@UR9FXE6QG>'  #Borda
```

Comment:
Instead of putting names here, can we maybe mention the full github ids as comments? So that we/other contributors also know whom to ping (we know, but other community members may not)?

Reply:
well, this needs t be Slack UID, so not even slack name nor gh nick as both can be changed, so it quite makes sense :)

Reply:
I know, but the github nick is the one that is easily visible for everyone using/contributing to this project :)

Reply:
ok, so you suggest, in addition to Slack mention, also adding GH nick?

Reply:
yep :)

Reply:
Just the comment in the end to say `#@akihironitta ` instead of `#Aki` :)

Reply:
done :+1: